primerpedia.js

```diff
@@ -24,22 +24,29 @@ var apiExtractsQuery = "action=query&prop=extracts&exintro&indexpageids=true&for
 var requestTimeoutInMs = 3000;
 var requestCallbackName = "requestCallback";
 
+var searchTermInputElement = null;
+var contentDivElement = null;
+var viewLinkElem = null;
+var editLinkElement = null;
+var articleTitleElement = null;
+var licenseIconElement = null;
+var infoIconElement = null;
+
 function random() {
-	document.getElementById('search-term').value = '';
+	searchTermInputElement.value = "";
```

Comment:
I'd normally prefer the '' --> "" change to be in the appropriate PR (#32), but this is quite minor, so let's keep it here for convenience :)

Reply:
I had the feeling something from another branch had slipped into this change. Sorry! :)

Reply:
It's understandable :) no problem.